packages/app-core/src/ui/App/ViewContainer.tsx

```diff
@@ -56,6 +56,18 @@ const ViewContainer = observer(function ({
     }
   }, [ref, session, view])
 
+  useEffect(() => {
+    // ensures there is always one view in focus, if there is a view open
+    if (
+      session.focusedViewId &&
+      session.views.some(view => view.id === session.focusedViewId)
+    ) {
+      session.setFocusedViewId(session.focusedViewId)
```

Comment:
unclear why this branch of the if statement is needed (seems like a do nothing?)

Reply:
It's to set the focus when the page has been refreshed or reloaded

"if there is a focused view Id and that view Id matches one of the views in the session, set it to focused"

Reply:
the code for setFocusedViewId is essentially

```
setFocusedViewId(id:number) {
  session.focusedViewId=id
}
```

so i think `session.setFocusedViewId(session.focusedViewId)` just assigns to itself, and could probably be uneeded?